### DISCUSSION TITLE
The Lustia tp Bug

### DISCUSSION THREAD

**Discussion by RandomGuy784** [ErinaSugino/Starbound-SBR-Mod#138] [Trouble Shooting] [answered]
I've tried countless times to enter Lustia, but I always crash, I follow what one discussion said and tried to manually download a file from the source on here, but it keeps coming up with this and idk how to fix it.
![Screenshot 2024-10-05 230902](https://github.com/user-attachments/assets/aa1c8deb-951f-44bb-a319-4d1c1b89c738)

**Comment by ErinaSugino** [comment]
From where exactly did you download it? Cause that is certainly the solution.

**Reply by RandomGuy784** [reply]
Downloaded it exactly from LoversLab. Source Version. (I changed to that from the pak version because I thought it would help solve it.)

**Reply by ErinaSugino** [reply, marked as answer]
Yeah no, what the other discussion tells you to do is download the specific file `/tilesets/packed/objects-special/sexscape_tileset.json` from here on GitHub. From the main branch itself, NOT the release files. And replace it in your local mod, which requires the source version.

**Reply by RandomGuy784** [reply]
Just did now, and it completely solved it. Thanks for the help.
